
### A collection of bioinformatics tools for use with [Galaxy](https://galaxyproject.org/) written at Quadram Institute

[![Build Status](https://travis-ci.com/quadram-institute-bioscience/galaxy-tools.svg?branch=master)](https://travis-ci.com/quadram-institute-bioscience/galaxy-tools)

#### Total: 36 tools
- **[AMRFinderPlus](https://github.com/ncbi/amr/)** NCBI Antimicrobial Resistance Gene Finder Plus
- **[abricate](https://github.com/tseemann/abricate)** Mass screening of contigs for antiobiotic resistance genes
- **[albatradis](https://github.com/quadram-institute-bioscience/galaxy-tools)** Comparative TraDIS analysis
- **[ariba](https://github.com/quadram-institute-bioscience/galaxy-tools)** Antimicrobial Resistance Identification By Assembly
- **[bin3c](https://github.com/cerebis/bin3C)** Extract metagenome-assembled genomes (MAGs) from metagenomic data using Hi-C
- **[biotradis](https://github.com/quadram-institute-bioscience/galaxy-tools)** A set of tools to analyse the output from TraDIS analyses
- **[bracken](https://github.com/quadram-institute-bioscience/galaxy-tools)** Bayesian Reestimation of Abundance with KrakEN
- **[canu](https://github.com/marbl/canu)** Canu is a hierarchical assembly pipeline designed for high-noise single-molecule sequencing (such as the PacBio RS II/Sequel or Oxford Nanopore MinION). Adapted from bgruening's wrapper
- **[checkm_lineage_wf](https://github.com/Ecogenomics/CheckM)** Assess the quality of microbial genomes recovered from isolates, single cells, and metagenomes -- Lineage-specific Workflow
- **[circlator](https://sanger-pathogens.github.io/circlator/)** A tool to circularize genome assemblies
- **[coverm](https://github.com/wwood/CoverM)** Read coverage calculator for metagenomics
- **[dfast](https://github.com/nigyta/dfast_core)** DDBJ Fast Annotation and Submission Tool
- **[fargene](https://github.com/fannyhb/fargene)** fARGene (Fragmented Antibiotic Resistance Gene iENntifiEr )
- **[fastani](https://github.com/ParBLiSS/FastANI)** Fast alignment-free computation of whole-genome Average Nucleotide Identity
- **[flye](https://github.com/fenderglass/Flye/)** Assembly of long and error-prone reads. Adapted from bgruening's wrapper
- **[gtdbtk](https://github.com/Ecogenomics/GTDBTk)** A toolkit for assigning objective taxonomic classifications to bacterial  and archaeal genomes
- **[iqtree](http://www.iqtree.org/)** Efficient phylogenomic software by maximum likelihood
- **[kraken2](https://github.com/quadram-institute-bioscience/galaxy-tools)** Kraken taxonomic sequence classification system
- **[krocus](https://github.com/quadram-institute-bioscience/galaxy-tools)** Predict MLST directly from uncorrected long reads
- **[krona](https://github.com/marbl/Krona/)** Interactively explore metagenomes and more from a web browser
- **[medaka](https://github.com/nanoporetech/medaka)** Create a consensus sequence from nanopore sequencing data
- **[metabat2](https://bitbucket.org/berkeleylab/metabat)** MetaBAT Metagenome Binning based on Abundance and Tetranucleotide frequency
- **[nanofilt](https://github.com/wdecoster/nanofilt)** Filtering and trimming of long read sequencing data
- **[nanostat](https://github.com/wdecoster/nanostat)** Calculate various statistics from a long read sequencing dataset in fastq, bam or albacore sequencing summary format
- **[nullarbor](https://github.com/tseemann/nullarbor)** Pipeline to generate complete public health microbiology reports from sequenced isolates
- **[pacbio](https://github.com/PacificBiosciences)** Pacbio Tool Set
- **[parsnp](https://harvest.readthedocs.io)** Rapid core genome multi-alignment
- **[plasmidtron](https://github.com/sanger-pathogens/plasmidtron)** Assembling the cause of phenotypes and genotypes from NGS data
- **[qc3c](https://github.com/cerebis/qc3C)** Quality control for Hi-C DNA sequencing libraries
- **[qiime_add_label](http://www.qiime.org)** QIIME to perform microbial community analysis
- **[rebaler](https://github.com/rrwick/Rebaler)** Conducts reference-based assemblies using long reads
- **[snippy](https://github.com/quadram-institute-bioscience/galaxy-tools)** Snippy finds SNPs between a haploid reference genome and your NGS sequence reads. Test version, please do not use.
- **[socru](https://github.com/quadram-institute-bioscience/galaxy-tools)** Order and orientation of complete bacterial genomes
- **[sortmerna](http://bioinfo.lifl.fr/RNA/sortmerna/)** (adapted from rnateam, don not use) SortMeRNA is a software designed to rapidly filter ribosomal RNA fragments from metatransriptomic data produced by next-generation sequencers.
- **[super_distance](https://github.com/quadram-institute-bioscience/super_distance)** Matrix Representation with Distances: calculates pairwise distances between gene leaves, and estimates species trees from summary distance matrices
- **[tiptoft](https://github.com/quadram-institute-bioscience/galaxy-tools)** Predict plasmids from uncorrected long read data
